vagrant_scripts/provision.sh

```diff
@@ -48,13 +48,14 @@ eval "$(pyenv virtualenv-init -)"
 pyenv install 3.6.8
 pyenv virtualenv 3.6.8 swarfarm-3.6.8
 pyenv activate swarfarm-3.6.8
```

Comment:
This activate line is technically no longer required since the next line activates the venv as part of the command

Reply:
The `pyenv deactivate` command further down should also be removed

Reply:
good catch, updated